### PR TITLE
added python code to cses-1644

### PR DESCRIPTION
**Pull request by SuryaBZ**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1644.mdx [hunk 1]
```diff
@@ -61,4 +61,33 @@ int main() {
...
+A = item[1]
+B = item[2]
+pfx = [0]*(N+1)
+for i in range(1,N+1):
+	a = int(input())
+	pfx[i] = a + pfx[i-1]	//Construct a prefix sum
+ret = -linf
+ms = []
+
+// we can keep a sliding window of size B, then find the lowest pfx[j] using list.sort()
```
Comment (SansPapyrus683) [inline solutions/cses-1644.mdx:80]: ...do you know how python comments work?

File: solutions/cses-1650.mdx [hunk 2]
```diff
@@ -111,4 +111,27 @@ public class RangeXorQueries {
 
 </JavaSection>
 
+<PySection>
+
+```py
+n = int(input())
+q = int(input())
+
+arr = [0] * (n + 2)
```
Comment (SansPapyrus683) [inline solutions/cses-1650.mdx:120]: why n+2? does n+1 work?

File: solutions/cses-1644.mdx [hunk 3]
```diff
@@ -57,8 +57,37 @@ int main() {
...
+
+```py
+linf = 1e18
+N, A, B = list(map(int, input().split()))
+
+pfx = [0] * (N + 1)
+# Construct our prefix sum array
+for i in range(1, N + 1):
+	a = int(input())
+	pfx[i] = a + pfx[i-1]
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1644.mdx:74]:
```suggestion
	pfx[i] = a + pfx[i - 1]
```

File: solutions/cses-1644.mdx [hunk 4]
```diff
@@ -57,8 +57,37 @@ int main() {
...
+
+ret = -linf
+ms = []
+# we can keep a sliding window of size B, then find the lowest pfx[j] using list.sort()
+for i in range(A, N + 1):
+	# Remove the last element if our size is above B.
+	if i > B:
+		ms.remove(pfx[i - B - 1])
+	ms.append(pfx[i-A])
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1644.mdx:83]:
```suggestion
	ms.append(pfx[i - A])
```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

File: solutions/cses-1644.mdx [hunk 5]
```diff
@@ -57,8 +57,34 @@ int main() {
 		ret = max(ret, pfx[i]-*ms.begin()); //we want to minimize ms.begin() aka pfx[j]
 	}
 
-	cout << ret << "\n";
+	cout << ret << endl;
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1644.mdx:60]:
```suggestion
	cout << ret <<"\n";
```
keep it as \n.

**Review by Aadit-Ambadkar — APPROVED** [review]

**Pull request closed without merging**